symbols.js

```diff
@@ -491,11 +638,44 @@ document.addEventListener("DOMContentLoaded", () => {
     });
     searchInput.addEventListener("blur", (e) => {
         window.location.hash = e.target.value;
+		return false;
     });
 
     window.addEventListener("hashchange", () => {
         const search = window.location.hash ? window.location.hash.substring(1) : "";
         searchInput.value = search;
         renderSymbols(search);
     });
+    
+    document.getElementById("add_symbol").addEventListener("click", () => addSymbol());
+    
+    window.addEventListener("dblclick", (e) => {
+        let target = e.target;
+        while(target) {
+			console.log(target);
+            if (target.classList?.contains("remove")) {
+                return removeSymbol(target.parentElement);
+            }
+            if (target.classList?.contains("glyph")) {
+                return editSymbol(target.parentElement, "glyph");
+            }
+            if (target.classList?.contains("name")) {
+                return editSymbol(target.parentElement, "name");
+            }
+            if (target.classList?.contains("copy")) {
+                return editSymbol(target.parentElement, "name");
+            }
+            if (target.classList?.contains("symbol")) {
+                return editSymbol(target, "glyph");
+            }
+            target = target.parentElement;
+        }
+    });
+	
+	window.addEventListener("dragstart", handleDragStart);
+	window.addEventListener("dragover", handleDragOver);
+	window.addEventListener("dragenter", handleDragEnter);
+	window.addEventListener("dragleave", handleDragLeave);
```

Comment:
Can likely remove this, the handler for it is empty.

Reply:
dragover needs to be handled, else the elements are not droppable
dragleave can be removed and is in 914415c9dd6387c7983bcf6ddd23508b77b5c2fe
other handlers are not empty, they stay.